packages/manager/modules/support/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@ovh-ux/manager-support",
+  "version": "0.0.0",
+  "license": "BSD-3-Clause",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/support.js",
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
+    "dev": "rollup -c --environment BUILD:development",
+    "dev:watch": "yarn run dev --watch",
+    "prepare": "yarn run build",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-support' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-support' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-support' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "devDependencies": {
+    "@ovh-ux/component-rollup-config": "^6.0.0"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^6.1.2",
+    "@ovh-ux/ng-ovh-otrs": "^7.1.7",
+    "@ovh-ux/ng-translate-async-loader": "^2.0.0",
+    "@uirouter/angularjs": "^1.0.22",
+    "angular": "^1.7.8",
+    "angular-translate": "^2.18.1",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
```

Comment:
This is more a `deps` than a peerDeps. 
If you want to integrate this to `manager-dedicated`, you currently will have trouble with `lodash` for example